src/Components/Alert/Alert_Definer.php

```diff
@@ -12,9 +12,24 @@
 
 class Alert_Definer implements Definer_Interface {
 
+	public const COLOR_OPTIONS = 'alert.color_options';
+
 	public function define(): array {
 		return [
-			Alert_Rule_Manager::class => DI\autowire()
+			Color_Options_Manager::class => Alert_Color_Options::class,
+			self::COLOR_OPTIONS          => DI\add( [
```

Comment:
Know what, let's just go ahead and add an `apply_filters( 'tribe/alerts/color_options' )` inside the `DI\add()` and update the README to include that filter and example of providing your own custom colors.